### ISSUE TITLE
Cifar-10 training loss

### ISSUE THREAD

**Issue by slala2121** [google-research/simclr#132]
Should the training loss for both the contrastive learning and supervised converge to 0?

**Comment by chentingpc** [comment]
The cross entropy loss could theoretically go to zero (if the model fully
overfits all data), but it is unlikely to happen in practice (especially
with augmentation); however, the weight decay loss won't go to zero at the
same time, otherwise all weights are zero.

**Comment by slala2121** [comment]
Thanks for your reply. 

Ok. I am only pretraining the model. Here are sample loss curves (for different learning rates) that I'm seeing on 1K samples taken from cifar10. The total contrast_loss appears to saturate about 4000 steps to around 9 (weight decay is set to 0), while the contrast entropy increases rapidly. 

Do these seem resemblant of the loss curves you found in your experiments?

![Screen Shot 2021-05-23 at 7 54 35 AM](https://user-images.githubusercontent.com/58089750/119266018-ecadba00-bb9d-11eb-8106-4bc3a4d2fb2a.png)

**Comment by chentingpc** [comment]
the trend looks fine to me.

**Comment by slala2121** [comment]
Great, thanks.
